app/Resources/docs/installing.md

```diff
@@ -8,8 +8,11 @@
 ##Installation
 1. Download a copy of an Overwatch release [here](https://github.com/zsturgess/overwatch/releases/latest) (You should be downloading the copy that has a filename like `overwatch-VERSION.tar.gz`) and unpack the project in a location of your choosing, ensuring the web server's document root is set to the web/ folder.
 2. Set up a database for overwatch to use. (Overwatch expects a MySQL database by default, although you can change the driver for doctrine in the next step)
-3. Edit the details in the `app/config/parameters.yml` file, or alternatively provide them as environment variables prefixed with `OVERWATCH_` - for example `OVERWATCH_MAILER_FROM` 
-4. Run `php app/console doctrine:schema:create` to initialise the database.
+3. Create a copy of `app/config/parameters.yml.dist`, in `app/config/parameters.yml`
+4. Edit the details in the `app/config/parameters.yml` file, or alternatively provide them as environment variables prefixed with `OVERWATCH_` - for example `OVERWATCH_MAILER_FROM` 
+5. Run `composer install` (assuming you have a global Composer installation) to install all the PHP dependencies.
```

Comment:
The specially prepared .tar.gz file available for download from the releases page has both pre-copied the parameters.yml.dist file to parameters.yml and run a composer install --no-dev, so steps 3 and 5 should not be required.